backend/src/api/users.ts

```diff
@@ -135,11 +135,13 @@ export const update = wrapHandler(async (event) => {
     }
   );
   if (user) {
+    console.log(JSON.stringify({ original_user: user }));
```

Comment:
@actualeyes should we add something like `"/users/{id}: Updating user from {original user} to {user}"` in the log as well? So we can have some kind of standardized format that's easy to search, so we know which endpoint calls which piece of code.